source/transactions/transactions.rst

```diff
@@ -150,6 +156,17 @@ This section is an overview of the public API for transactions:
          * The readPreference to use for this transaction.
          */
         Optional<ReadPreference> readPreference;
+
+        /**
+         * The maximum amount of time to allow the *commit* to run.
```

Comment:
Does anyone else think it may be misleading for this option to be named `maxTimeMS`. That seems to imply that it is a limit for the _entire transaction_ but it only applies to the _commit_.

I think we could improve this in one of two ways:
1. We can ask the server to support `maxTimeMS` on the initial `startTransaction: true` command. This way the server can apply the `maxTimeMS` limit the lifetime of the entire transaction similar to how `maxTimeMS` works on a cursor.
2. We can rename this option to make it more clear that it only applies to the commit, like `maxCommitTimeMS`.

Note I considered adding a `maxTimeMS` parameter to ClientSession.commitTransaction and rejected that idea because it would be inconsistent with writeConcern and it would also make it impossible to use maxTimeMS with the `withTransaction` API where the commit call is implicit.

Any thoughts?